<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActorList</class>
 <widget class="QWidget" name="ActorList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QWidget" name="settingsHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QLabel" name="settingLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Actors</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>280</width>
     <height>135</height>
    </rect>
   </property>
   <widget class="QPushButton" name="actor1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Actor 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="actor2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>35</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Actor 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="actor3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Actor 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="actor4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>105</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Actor 4</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="pageHead" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_right_white.svg</normaloff>:/settings/icons/arrow_right_white.svg</iconset>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="page">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Page 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
